src/messages.tsx

```diff
@@ -0,0 +1,20 @@
+export const NO_CONTEXTS_SAVED = 'No contexts saved yet, run asyncapi --help to know more';
+
+export const CONTEXT_NOT_FOUND = 'This context key does not exists.';
+
+export const KEY_NOT_FOUND = 'Key not found';
+
+export const DELETING_CURRENT_CONTEXT = 'You are tyring to delete a context that is currently in use.';
+
+export const MISSING_CURRENT_CONTEXT = 'No context is set as current, please set a current context.';
+
+export const MISSING_ARGUMENTS = 'Missing arguments.';
+
+export const NEW_CONTEXT_ADDED = 'New context added.';
```

Comment:
How about?

`New context added. \nYou can set it as your current context:\n      asyncapi context use {CONTEXT_NAME}\nYou can use this context when needed with --context flag`

Reply:
@derberg I was making these changes, and I think It would look good if the command example has a different color, but it is hard to make them in different colors in a string. But as we are using ink I think making all the text messages a component would be nice.

Reply:
yup, definitely, one of the reasons we use Ink. I guess that you would prefer to do it in a followup PR to not complicate this one?

Reply:
Ok so in this PR I will just keep it as it is and update the texts. Then in another follow-up pr, we can convert all string constants to components.